### ISSUE TITLE
Vertical slider doesn't announce min/max with VoiceOver and JAWS

### ISSUE THREAD

**Issue by jelbourn** [angular/components#11134]
### Summary
vertical slider does not announce min / max value while value being increased / decreased. 

### Reproduction on material.angular.io
1. Linear navigate to ' Components ' menu item and click 'Enter' to open components page.                                                                                                                                                                                                                                          
2. Linear navigate forward and set focus on expand collapse left hand menu item ' Form Controls'                                                                                                                                                                                                                                  
3. Click 'Enter' key to expand OR collapse this menu item                                                                                                                                                                                                                                                                                       
4. Linear navigate into "Slider' & click ENTER keystroke.                                                                                                                                                                                                                                                                                   
5. Navigate into tabs "Examples" and Click ENTER                                                                                                                                                                                                                                                                                 
6. Navigate into  "show ticks" and click space bar to check the checkbox                                                                                                                             
7. navigate forward into ' tick interval stepper and click to select                                                                                                                               
8. navigate to vertical check box and click to select                                                                                                                                                             
9. navigate further and set focus into the slider and press arrow keys to increase / decrease the slider


### Additional comments
ChromeVox behaviour is as expected but VO & Jaws only read the value without reading the min / max value.

**Comment by jelbourn** [comment]
This is potentially fixed in MDC, _but_ the mdc-based slider doesn't yet support vertical mode.

**Comment by wagnermaciel** [comment]
Closing as the feature request for a vertical slider has been closed as "won't fix"
https://github.com/angular/components/issues/26695
